web-animations-1/Overview.bs

```diff
@@ -4580,8 +4624,16 @@ dictionary OptionalEffectTiming {
     time taken to complete a single iteration of the <a>animation
     effect</a>.
 
-    In this level of this specification, the string value <code>auto</code>
-    is treated as the value zero for the purpose of timing model calculations
+    If the <a lt="timeline duration">duration</dfn> of the Animation's
+    <a>timeline</a> is a percentage, a non <code>auto</code> duration will
```

Comment:
Since you _just_ required the duration to be a percentage, do you need to mention non-auto?

Reply:
I just required the timeline duration to be a percentage, however the non-auto duration refers to the animation duration.